.github/workflows/ci.yml

```diff
@@ -28,6 +28,10 @@ jobs:
           ~/.cargo/git
           backend/target
         key: ${{ runner.os }}-cargo-${{ hashFiles('backend/cache-fingerprint') }}
+    - name: Add dummy frontend assets
+      run: |
+        mkdir ../frontend/build
+        touch ../frontend/build/index.html
```

Comment:
Why not build the frontend first and use the actual assets here? :thinking:

Reply:
I already fixed this on another branch :D 
I merge the two big CI steps. Will create a PR once this is merged.